third_party/move/move-prover/move-docgen/src/docgen.rs

```diff
@@ -719,6 +719,118 @@ impl<'env> Docgen<'env> {
         }
     }
 
+    fn gen_spec_table(input: &str) -> String {
+        let blocks = input.split("\n\n").collect::<Vec<_>>();
+        let mut html_table = String::from(
```

Comment:
If time permits, maybe you could optimize the function a little more, for instance, write a function that takes a list of table row names, and an input string, as the input and generate a html table?

Reply:
Done! Thanks.